itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationWithTableMigration.java

```diff
@@ -504,4 +507,56 @@ public void dynamicallyConvertExternalToManagedTable() throws Throwable {
             .runFailure("alter table t1 set tblproperties('EXTERNAL'='false')")
             .runFailure("alter table t2 set tblproperties('EXTERNAL'='false')");
   }
+
+  @Test
+  public void testMigrationWithUpgrade() throws Throwable {
+    WarehouseInstance.Tuple tuple = primary.run("use " + primaryDbName)
+            .run("create table tacid (id int) clustered by(id) into 3 buckets stored as orc ")
```

Comment:
Better to have some data in tacid table too. To know if writeId seeding impacts data read after bootstrap.

Reply:
done